lib/hermetic/youtrack.ex

```diff
@@ -59,32 +59,40 @@ defmodule Hermetic.YouTrack do
     base_url() <> "/static/apple-touch-icon-180x180.png"
   end
 
-  @doc ~S"""
-  Fetch list of all available YouTrack project IDs.
-  """
-  @spec project_ids :: list(String.t())
-  def project_ids do
-    json = Jason.decode!(request(:get, "/rest/project/all").body)
-    for %{"shortName" => id} <- json, do: id
+  defmodule ProjectID do
+    use LambdaCache, name: __MODULE__
+
+    @doc ~S"""
+    Fetch list of all available YouTrack project IDs.
+    """
+    @spec refresh :: list(String.t())
+    def refresh do
+      json = Jason.decode!(YouTrack.request(:get, "/rest/project/all").body)
+      for %{"shortName" => id} <- json, do: id
+    end
   end
 
   @spec cached_project_ids :: list(String.t())
   def cached_project_ids do
-    Cache.get(YouTrack.ProjectIDs)
+    ProjectID.retrieve(ProjectID)
```

Comment:
Same as above. Not sure module should know some of its functions are cached. Maybe remove that function and call cache directly?